e2e_tests/integration/stepDefinitions/index.ts

```diff
@@ -226,16 +222,62 @@ When('Maria confirms the existing custody status', () => {
+  expect(contents).to.match(data.cro as RegExp)
+  expect(contents).to.match(data.pnc as RegExp)
+  expect(contents).to.match(data.prisonNo as RegExp)
+  expect(contents).to.match(data.noms as RegExp)
+}
+export const q5SentenceDetails = (contents: string) => {
+  expect(contents).to.match(data.indexOffence as RegExp)
+  expect(contents).to.match(data.dateOfOriginalOffence as RegExp)
+  expect(contents).to.match(data.dateOfSentence as RegExp)
+  expect(contents).to.match(data.lengthOfSentence as RegExp)
+  expect(contents).to.match(data.licenceExpiryDate as RegExp)
+  expect(contents).to.match(data.sentenceExpiryDate as RegExp)
+  expect(contents).to.match(data.custodialTerm as RegExp)
+  expect(contents).to.match(data.extendedTerm as RegExp)
+}
+export const q6CustodyStatus = (contents: string, answer: string) =>
+  expect(contents).to.contain(`Is the offender currently in police custody or prison custody? ${answer}`)
+export const q7Addresses = (contents: string, answer: string) =>
+  expect(contents).to.contain(`If the offender is in police custody, state where: ${answer}`)
+export const q8ArrestIssues = (contents: string, answer: string, details: string) => {
   expect(contents).to.contain(`Are there any arrest issues of which police should be aware?  ${answer}`)
-  expect(contents).to.contain(`If yes, provide details below, including information about any children or vulnerable adults linked to any of the above addresses: ${details}`)
+  expect(contents).to.contain(
+    `If yes, provide details below, including information about any children or vulnerable adults linked to any of the above addresses: ${details}`
+  )
+}
+export const q12MappaDetails = (contents: string) => {
+  expect(contents).to.match(data.mappaCategory as RegExp)
+  expect(contents).to.match(data.mappaLevel as RegExp)
 }
-export const assertQ22_recall_type = (contents: string, answer: string, details: string) => {
+export const q16IndexOffenceDetails = (contents: string) => {
+  // FIXME: For this to work on dev and preprod, we need a corresponding record in OASys for our CRN
+  expect(contents).to.match(data.indexOffenceDetails as RegExp)
+}
+export const q22RecallType = (contents: string, answer: string, details: string) => {
   expect(contents).to.contain(`Select the proposed recall type, having considered the information above: ${answer}`)
   expect(contents).to.contain(`Explain your reasons for the above recall type recommendation: ${details}`)
 }
+export const q25ProbationDetails = (contents: string) => {
+  // FIXME: Name of our user not getting pulled through
+  expect(contents).to.match(data.nameOfPersonCompletingForm as RegExp)
+  expect(contents).to.match(data.emailAddressOfPersonCompletingForm as RegExp)
+  expect(contents).to.match(data.region as RegExp)
+  expect(contents).to.match(data.ldu as RegExp)
+  expect(contents).to.contain(`${data.dateOfDecision} ${DateTime.now().toFormat('dd/MM/y')}`)
+  expect(contents).to.match(data.timeOfDecision as RegExp)
```

Comment:
Do you not want to check for the time of the decision as a regex as well?

Reply:
checking the time might be fragile; if the timestamp is taken in the last few seconds of a minute, and the test generates a timestamp, they might not match

Reply:
yeh, hence the reason I'm saying can we check as a regex? e.g. ^([0-1]?[0-9]|2[0-3]):[0-5][0-9]$

Reply:
done